md-setup/src/main/java/com/wipro/ats/bdre/md/setup/SetupDB.java

```diff
@@ -162,11 +163,13 @@ private void populateBatch(String dataFile) throws Exception {
 
             }
 
-            //session.flush();;
+
         } catch (Exception e) {
-            LOGGER.error("In File: " + dataFile + "; Bad Line: " + line);
+            String inFile="In File: ";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "In File: " 16 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)